### ISSUE TITLE
Update Release Notes For Important Change in triplet_semihard_loss

### ISSUE THREAD

**Issue by DrChrisLevy** [tensorflow/addons#2095]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 18.04):
- TensorFlow version and how it was installed (2.3 pip in docker):
- TensorFlow-Addons version and how it was installed (pip in docker, tensorflow-addons==0.11.1):
- Python version: 3.6 
- Is GPU used? (no):

**Describe the bug**
Not so much a bug. But this change could have been better documented in the release notes.
I had a unit test and it caught this when I upgraded from 0.10.0 to 0.11.1.

0.11.1. VERSION: https://github.com/tensorflow/addons/blob/v0.11.1/tensorflow_addons/losses/triplet.py#L123

0.10.0 VERSION: https://github.com/tensorflow/addons/blob/v0.10.0/tensorflow_addons/losses/triplet.py#L128

Basically the default value for argument `squared` was changed from `True` to `False`. To achieve the old behaviour you must use the new arg `squared-L2`.

**Code to reproduce the issue**
Not needed as clearly showed the git diff above between the two versions.

**Other info / logs**
Maybe this change could be documented in the change log, https://github.com/tensorflow/addons/releases/tag/v0.11.0, under tfa.losses section. It might help others if all of a sudden their model training is behaving differently. 

Thanks!

**Comment by WindQAQ** [comment]
Thanks for pointing out this! I've updated the release note. Please take a look at it to see if it works well :-) Thank you!

**Comment by DrChrisLevy** [comment]
Perfect. That’s great. Thanks for the fast response.
